### PR TITLE
Options request one

### PR DESCRIPTION
**Pull request by grace000**
This PR addresses this story.

- Client can make an options request 
- ~~Also added enums for status codes~~

### CODE REVIEW

File: src/main/java/github/grace000/erversay/Constants/Headers.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package github.grace000.erversay.Constants;
+
+public final class Headers {
+    public final static String PIGGLY_BODY = "piggly wiggly";
```
Comment (ksabbak) [inline src/main/java/github/grace000/erversay/Constants/Headers.java]: Does this really belong in the `Constants/Headers` file? It doesn't seem like a header...

File: src/main/java/github/grace000/erversay/RequestRouter.java [hunk 2]
```diff
@@ -22,27 +29,33 @@ private Response getResponse(String path) {
             case PIGGLY_URI:
                 getResponseForPigglyGet();
                 break;
+            case METHODS_ONE_URI:
+                getResponseForOptions();
+                break;
             default:
                 getResponseForNotFound();
```
Comment (ksabbak) [inline src/main/java/github/grace000/erversay/RequestRouter.java:32]: If this is the default response from your server now, do you think that `DEFAULT_STATUS` is a good name for your 200 code?

**Comment by ksabbak** [comment]
Why are you merging into your head-request-tests branch?

File: src/main/java/github/grace000/erversay/Constants/StatusCodes.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package github.grace000.erversay.Constants;
+
+public final class StatusCodes {
+    public final static String DEFAULT_STATUS = "200 OK";
+    public final static String NOT_FOUND_STATUS = "404 Not Found";
+}
+
+
```
Comment (ssunday) [inline src/main/java/github/grace000/erversay/Constants/StatusCodes.java]: extra spaces

File: src/main/java/github/grace000/erversay/RequestRouter.java (hunk 2 above)
Comment (grace000) [reply src/main/java/github/grace000/erversay/RequestRouter.java:32]: That's not a good name if only certain requests will get a 200 code. The default might actually be a 404. I'll rename that to be more descriptive.

**Comment by grace000** [comment]
Re: merging into head-request-tests

I would merge into master, but this is my attempt at making sure you don't see the old code that you've already made comments for in the multiple PRs.

File: src/main/java/github/grace000/erversay/Constants/StatusCodes.java (hunk 3 above)
Comment (grace000) [reply src/main/java/github/grace000/erversay/Constants/StatusCodes.java]: Where? Within the string?

Comment (ssunday) [reply src/main/java/github/grace000/erversay/Constants/StatusCodes.java]: end of file

**Pull request merged**